integration/nwo/fabric/network/checks.go

```diff
@@ -0,0 +1,218 @@
+/*
```

Comment:
no idea why this is part of the PR

Reply:
I had to modify the CheckTopology function to support HSMX509Identities.
Now, the function was too big and I took the chance to modularize it.